homeassistant/components/swidget/coordinator.py

```diff
@@ -37,9 +43,9 @@ def __init__(
             ),
         )
 
-    async def async_initialize(self) -> Any:
+    async def async_initialize(self) -> bool:
         """Initialize a callback for any websocket events received from the device."""
-        return self.device.add_event_callback(self.websocket_event_callback)
+        return cast(bool, self.device.add_event_callback(self.websocket_event_callback))
```

Comment:
The problem is that the package is missing a return type annotation, can't that be fixed instead of casting here?

I mean, change the package:

```diff
-    def add_event_callback(self, callback: Callable[[Any], Any]):
+    def add_event_callback(self, callback: Callable[[Any], Any]) -> bool:
```